src/sentry/static/sentry/app/views/settings/components/forms/model.jsx

```diff
@@ -325,18 +324,9 @@ class FormModel {
    */
   @action
   saveForm() {
-    // Represents state of current form
-    let form = this.getData();
+    Array.from(this.fieldDescriptor.keys()).forEach(id => !this.updateErrorState(id));
```

Comment:
There's a `validate` method with a TODO, can we move this into a `validateForm` and change `updateErrorState` to `validateField`?

Reply:
Yeah that sounds good